src/utils.cc

```diff
@@ -849,7 +849,11 @@ path resolve_path(const path& pathname)
   path temp = pathname;
   if (temp.string()[0] == '~')
     temp = expand_path(temp);
+#if (BOOST_VERSION >= 106000)
+  temp.lexically_normal();
```

Comment:
`lexically_normal` was introduced in Boost 1.60.0 (see [**New Features** section of Boost 1.60.0 Release Notes](https://www.boost.org/users/history/version_1_60_0.html)) so as long as Ledger maintains support for Boost <=1.60.0 (currently [1.49](https://github.com/ledger/ledger/blob/master/CMakeLists.txt#L89)) use of `lexically_normal` needs to be version guarded.